action.yml

```diff
@@ -37,9 +37,9 @@ runs:
         # determines the overall exit code and not the first failing.
         set +o pipefail
         # Try running the eslint command, so we'll fail if it can't run.
-        $(npm bin)/eslint --version
+        npm exec eslint --version
```

Comment:
```suggestion
        npm exec eslint -- --version
```
Jeg _tror_ der skal en `--` for at skelne argumenter til `eslint` fra argumenter til `npm`.

Reply:
Det tror jeg du har ret i jvf. docs